fabric-object-builder-api-v1/build.gradle

```diff
@@ -1,11 +1,19 @@
 archivesBaseName = "fabric-object-builder-api-v1"
-version = getSubprojectVersion(project, "1.2.0")
+version = getSubprojectVersion(project, "1.3.0")
 
 dependencies {
 	compile project(path: ':fabric-api-base', configuration: 'dev')
 	compile project(path: ':fabric-tool-attribute-api-v1', configuration: 'dev')
+
+	testmodImplementation project(path: ':fabric-tag-extensions-v0', configuration: 'dev')
+	testmodImplementation project(path: ":fabric-resource-loader-v0", configuration: 'dev')
 }
 
 minecraft {
 	accessWidener = file("src/main/resources/fabric-object-builder-api-v1.accesswidener")
 }
+
+tasks.withType(net.fabricmc.loom.task.AbstractRunTask) {
+	classpath += sourceSets.testmod.runtimeClasspath
+	classpath += sourceSets.testmod.output
+}
```

Comment:
Hmm, what does this do?

Reply:
so that `runClient` includes these test mods. Otherwise it will only include the main mod and will fail because project dependencies aren't added to classpath transitively (dunno why that is the case, but it happens)

Reply:
maybe we need to look into a `runTestClient/Server` task just for fabric api. Though not immediately a big issue

Reply:
yeah somehow my run task in ide doesn't work so i have to resort to gradle

Reply:
No, this is wrong. This needs sorting in another PR, fabric will need a `runTestClient/Sever` task. Not something that should be done like this.

Reply:
Also if something is wrong with your IDE task that needs fixing? What IDE?

Reply:
well, you can just try gradle fabric-object-builders-api-v1:runClient  , it currently fails

Reply:
also when intellij imports it will have the run client and server tasks but it will only have loader on classpath, nothing else

Reply:
Since the existing `fabric-object-builders-api-v1:runClient` does not run **at all** (just try in fabric main branch for yourself), there is no point in reverting this change for now. Can be fixed when run test configurations are added to add modules in fabric.

Reply:
you can edit the idea runconfigs to use the test module and it will work

Reply:
It doesn't, using the test module still doesn't include testmods